### PR TITLE
Init Polish localization

### PR DESCRIPTION
**Pull request by mfilocha**
An initial config for Polish localization.

### CODE REVIEW

**Comment by mfilocha** [comment]
A related PR to open Polish Slack channel: https://github.com/kubernetes/community/pull/4348

File: README-pl.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Więcej informacji na temat współpracy przy tworzeniu dokumentacji znajdziesz na stronach:
+
+* [Start contributing](https://kubernetes.io/docs/contribute/start/)
+* [Staging Your Documentation Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [Using Page Templates](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Documentation Style Guide](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Localizing Kubernetes Documentation](https://kubernetes.io/docs/contribute/localization/)
```
Comment (sftim) [inline README-pl.md]: You probably know, but these aren't yet localized.

File: README-pl.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## Różne wersje językowe `README.md`
+|  |  |
+|---|---|
+|[French README](README-fr.md)|[Korean README](README-ko.md)|
+|[German README](README-de.md)|[Portuguese README](README-pt.md)|
+|[Hindi README](README-hi.md)|[Spanish README](README-es.md)|
+|[Indonesian README](README-id.md)|[Chinese README](README-zh.md)|
+|[Japanese README](README-ja.md)|[Polish README](README-pl.md)|
```
Comment (sftim) [inline README-pl.md]: You probably know, but these aren't yet localized.

File: config.toml [hunk 3]
```diff
@@ -286,3 +286,14 @@ time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
 
+[languages.pl]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
```
Comment (sftim) [inline config.toml]: nit: This line is still English.

**Comment by sftim** [comment]
Another thing: I recommend creating a new branch for the initial Polish localization; see https://kubernetes.io/docs/contribute/localization/#branching-strategy

**Comment by remyleone** [comment]
@nvtkaszpir

File: README-pl.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+# Dokumentacja Kubernetes
+
+[![Build Status](https://api.travis-ci.org/kubernetes/website.svg?branch=master)](https://travis-ci.org/kubernetes/website)
+[![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
+
+Witamy!
+
+W tym repozytorium znajdziesz wszystko, co potrzebujesz do zbudowania [strony internetowej Kubernetes wraz z dokumentacją](https://kubernetes.io/). Bardzo nam miło, że chcesz wziąć w tym udział!
```
Comment (nvtkaszpir) [inline README-pl.md]: nit picking: `wszystko, czego potrzebujesz`

File: config.toml (hunk 3 above)
Comment (mfilocha) [reply config.toml]: Almost all other localizations keep this slogan untranslated. I'm trying to be consistent here.

File: README-pl.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+## Twój wkład w dokumentację
+
+Możesz kliknąć w przycisk **Fork** w prawym górnym rogu ekranu, aby stworzyć kopię tego repozytorium na swoim koncie GitHub. Taki rodzaj kopii (odgałęzienia) nazywa się *fork*. Zmieniaj w nim, co chcesz, a kiedy będziesz już gotowy/a przesłać te zmiany do nas, przejdź do swojej kopii i stwórz nowy *pull request*, abyśmy zostali o tym poinformowani.
```
Comment (nvtkaszpir) [inline README-pl.md]: Maybe it's worth to create additional page for abbreviations/disambiguation for such phrases as `fork`, `pull request` and so on?

File: README-pl.md (hunk 1 above)
Comment (mfilocha) [reply README-pl.md]: Yes, I know.

File: README-pl.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+## Jak uruchomić serwis internetowy lokalnie przy pomocy Dockera
+
+Zalecaną metodą uruchomienia serwisu internetowego Kubernetes lokalnie jest użycie specjalnego obrazu [Dockera](https://docker.com), który zawiera generator stron statycznych [Hugo](https://gohugo.io).
+
+> Użytkownicy Windows będą potrzebowali dodatkowych narzędzi, które mogą zainstalować przy pomocy [Chocolatey](https://chocolatey.org). `choco install make`
+
+> Jeśli wolisz uruchomić serwis lokalnie bez Dockera, przeczytaj [jak uruchomić serwis lokalnie przy pomocy Hugo](#jak-uruchomić-serwis-lokalnie-przy-pomocy-hugo) poniżej.
+
+Jeśli [zainstalowałeś i uruchomiłeś](https://www.docker.com/get-started) już Dockera, zbuduj obraz `kubernetes-hugo` Dockera lokalnie:
```
Comment (janisz) [inline README-pl.md]:
```suggestion
Jeśli [zainstalowałeś i uruchomiłeś](https://www.docker.com/get-started) już Dockera, zbuduj obraz `kubernetes-hugo` lokalnie:
```

File: README-pl.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+Po zbudowaniu obrazu, możesz uruchomić serwis lokalnie:
+
+```bash
+make docker-serve
+```
+
+Otwórz w przeglądarce adres http://localhost:1313, aby obejrzeć zawartość serwisu. Po każdej zmianie plików źródłowych, Hugo automatycznie aktualizuje stronę i odświeża jej widok w przeglądarce.
+
+## Jak uruchomić serwis lokalnie przy pomocy Hugo
```
Comment (janisz) [inline README-pl.md]:
```suggestion
## Jak uruchomić serwis lokalnie przy pomocy Hugo?
```

File: README-pl.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+|  |  |
+|---|---|
+|[French README](README-fr.md)|[Korean README](README-ko.md)|
+|[German README](README-de.md)|[Portuguese README](README-pt.md)|
+|[Hindi README](README-hi.md)|[Spanish README](README-es.md)|
+|[Indonesian README](README-id.md)|[Chinese README](README-zh.md)|
+|[Japanese README](README-ja.md)|[Polish README](README-pl.md)|
+|||
+
+## Jak uruchomić serwis internetowy lokalnie przy pomocy Dockera
```
Comment (janisz) [inline README-pl.md]:
```suggestion
## Jak uruchomić serwis internetowy lokalnie przy pomocy Dockera?
```

File: README-pl.md (hunk 5 above)
Comment (janisz) [reply README-pl.md]: We can start with this: https://github.com/nurkiewicz/polski-w-it

File: README-pl.md (hunk 2 above)
Comment (mfilocha) [reply README-pl.md]: What do you mean? I see the code in 1.17 branch.

Comment (sftim) [reply README-pl.md]: What I mean is that “French”, “Korean” etc are written in English not Polish.

File: config.toml (hunk 3 above)
Comment (sftim) [reply config.toml]: > Almost all other localizations keep this slogan untranslated.

That's true, but it's a bug. The slogan should be localized.

**Comment by sftim** [comment]
/language pl
?

File: README-pl.md (hunk 8 above)
Comment (nvtkaszpir) [inline README-pl.md]: `Jak wygenerować dokumentację lokalnie` feels a bit less awkward, don't you think?

File: README-pl.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+Zalecaną metodą uruchomienia serwisu internetowego Kubernetes lokalnie jest użycie specjalnego obrazu [Dockera](https://docker.com), który zawiera generator stron statycznych [Hugo](https://gohugo.io).
+
+> Użytkownicy Windows będą potrzebowali dodatkowych narzędzi, które mogą zainstalować przy pomocy [Chocolatey](https://chocolatey.org). `choco install make`
```
Comment (nvtkaszpir) [inline README-pl.md]: `choco install make` would be better wrapped in simple ```shell ``` block

Comment (mfilocha) [reply README-pl.md]: I know, this is how the original document is formatted.

**Comment by mfilocha** [comment]
Hi @sftim  
> For a localization typically you will need at least 2 contributors and some initial work. https://github.com/kubernetes/website/pull/12548/files shows an example of how the French localization started out, including a [`fr.toml`](https://github.com/remyleone/website/blob/3e9c4abe36131c45f68b3bcce84dfccffbc63838/i18n/fr.toml) for strings localization.

I'm aware of this requirements. I'm currently in search of authorized reviewers. 
And I created a minimum set of transtlation for version 1.16, which I'm going to update to 1.17 soon and start creating PRs:
 https://github.com/kubernetes/website/compare/release-1.16...mfilocha:dev-1.16-pl.1

File: README-pl.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+Zajrzyj do [oficjalnej dokumentacji Hugo](https://gohugo.io/getting-started/installing/) po instrukcję instalacji. Upewnij się, że instalujesz rozszerzoną wersję Hugo, określoną przez zmienną środowiskową `HUGO_VERSION` w pliku [`netlify.toml`](netlify.toml#L9).
+
+Aby uruchomić serwis lokalnie po instalacji Hugo, napisz:
+
+```bash
+make serve
+```
+
+Zostanie uruchomiony lokalny serwer Hugo na porcie 1313. Otwórz przeglądarkę na stronie http://localhost:1313, aby obejrzeć zawartość serwisu. Po każdej zmianie plików źródłowych, Hugo automatycznie aktualizuje stronę i odświeża jej widok w przeglądarce.
```
Comment (nvtkaszpir) [inline README-pl.md]: `Otwórz w przeglądarce adres....`

File: README-pl.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+## Społeczność, listy dyskusyjne, uczestnictwo i wsparcie
+
+Zajrzyj na stronę [społeczności](http://kubernetes.io/community/), aby dowiedzieć się, jak możesz się zaangażować w jej działania.
+
+Możesz kontaktować się z gospodarzami projektu za pomocą:
+
+* [Slack-a](https://kubernetes.slack.com/messages/sig-docs)
```
Comment (nvtkaszpir) [inline README-pl.md]: maybe `komunikatora Slack`?

File: README-pl.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+* [List dyskusyjnych](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+
+### Zasady postępowania
+
+Udział w działaniach społeczności Kubernetes jest regulowany przez [Zasady postępowania Kubernetes](code-of-conduct.md).
```
Comment (nvtkaszpir) [inline README-pl.md]: `Kodeks postępowania`, I think Kubernetes word can be dropped here.

File: README-pl.md (hunk 5 above)
Comment (mfilocha) [reply README-pl.md]: There is no translation of *fork* there :(

Comment (mfilocha) [reply README-pl.md]: Neither *pull request*...

Comment (sftim) [reply README-pl.md]: If you want to, add some of those terms to https://kubernetes.io/docs/reference/glossary/?community=true

but then _every_ localization has to translate them.

File: README-pl.md (hunk 2 above)
Comment (mfilocha) [reply README-pl.md]: Again, I'm following common practice.

File: README-pl.md (hunk 9 above)
Comment (nvtkaszpir) [reply README-pl.md]: Do we need to stick to it to the letter? ;)

**Comment by nvtkaszpir** [comment]
> @nvtkaszpir Would you consider be included as a reviewer in this localization?

yes, please.

File: README-pl.md (hunk 2 above)
Comment (mfilocha) [reply README-pl.md]: Fixed.

File: README-pl.md (hunk 4 above)
Comment (mfilocha) [reply README-pl.md]: Fixed.

File: README-pl.md (hunk 9 above)
Comment (mfilocha) [reply README-pl.md]: Okay, let's make it simply better :)

File: README-pl.md (hunk 10 above)
Comment (mfilocha) [reply README-pl.md]: Fixed.

File: README-pl.md (hunk 11 above)
Comment (mfilocha) [reply README-pl.md]: Fixed.

File: README-pl.md (hunk 12 above)
Comment (mfilocha) [reply README-pl.md]: Fixed.

File: README-pl.md (hunk 8 above)
Comment (mfilocha) [reply README-pl.md]: You are right, but it depends on how much we want to follow the original text.

File: README-pl.md (hunk 1 above)
Comment (mfilocha) [reply README-pl.md]: I've translated these titles.

**Comment by rlenferink** [comment]
> 
> 
> > @nvtkaszpir Would you consider be included as a reviewer in this localization?
> 
> yes, please.

@mfilocha Every translation needs to have a minimum of 2 owners, otherwise when you submit a pull request nobody can approve except the sig-docs-maintainers, which doesn't make sense because they can't review. In this case please add @nvtkaszpir as well to the owners section.

**Comment by rlenferink** [comment]
@remyleone / @mfilocha why is the pull request opened against the 1.17 branch instead of the 'master' branch? This is the initial translation so we probably want to have this against our latest/master branch as well.

**Comment by rlenferink** [comment]
@nvtkaszpir you need to request [org membership](https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E) as well. For that you need to have 2 sponsors. I can sponsor you and probably @remyleone wants to sponsor you as well, right?

**Comment by remyleone** [comment]
I will sponsor you @nvtkaszpir :)

**Comment by mfilocha** [comment]
@sftim Waiting for the language label: https://github.com/kubernetes/test-infra/pull/15684

**Comment by jimangel** [comment]
This PR needs to be opened against master, not `dev-1.17`. Also, please squash the commits if possible. Let me know if you need assistance with anything! Thanks!

/hold

**Comment by remyleone** [comment]
@mfilocha Are there remaining open issues about this Pull Request? Otherwise, we can close this one and open a new one that targets master.

File: config.toml (hunk 3 above)
Comment (sftim) [reply config.toml]: I've sent in a bunch of pull requests to fix the other localizations.

File: README-pl.md (hunk 5 above)
Comment (wojciechka) [reply README-pl.md]: I think **Fork** is explained later in the first paragraph. As for pull request, I am not familiar with any good term for that in Polish.

Perhaps we could contribute some vocabulary back to https://github.com/nurkiewicz/polski-w-it ? 

I think translations of newer or less commonly used terms into Polish are a wider problem. From what I've seen closer to me, people just use the English terms where there's no good translation available. And I've seen lack of good translated terms in a lot of places.

I suspect we'll run into quite a few words I always had a hard time translating - such as "packaging".

I'm not sure what's the best option here, but I think this is something we may want to address early on and define how to handle it.

**Comment by mfilocha** [comment]
@remyleone @jimangel I'll keep this PR open for a while to resolve remaining topics and then I'll open a new PR against the master branch, as advised.

File: README-pl.md (hunk 8 above)
Comment (mfilocha) [reply README-pl.md]: What about: "Jak uruchomić lokalnie stronę z dokumentacją przy pomocy Dockera?"

Comment (wojciechka) [reply README-pl.md]: Based on English translation, the text is "Running the website locally using Docker". Earlier there's "(...)the Kubernetes website and documentation", which was translated as "strony internetowej Kubernetes wraz z dokumentacją".

Why not just "Jak uruchomić stronę internetową Kubernetes lokalnie przy pomocy Dockera"?

Comment (mfilocha) [reply README-pl.md]: @nvtkaszpir was not happy about the original translation.

File: config.toml (hunk 3 above)
Comment (kpucynski) [reply config.toml]: As discussed on slack #kubernetes-docs-pl:
```suggestion
description = "Produkcyjny system zarządzania kontenerami"
```

**Comment by mfilocha** [comment]
I created another PR: https://github.com/kubernetes/website/pull/18419 as advised by reviewers. This one is going to be closed today without merging.

**Comment by sftim** [comment]
/close
as per https://github.com/kubernetes/website/pull/18254#issuecomment-570251536

**Pull request closed without merging**